control_panel_api/tests/test_models.py

```diff
@@ -19,11 +18,11 @@
 class UserTestCase(TestCase):
     @patch('control_panel_api.services.create_role')
     def test_aws_create_role_calls_service(self, mock_create_role):
-        username = 'james'
-        user = User.objects.create(username=username)
+        auth0_id = 'github|1'
+        user = User.objects.create(auth0_id=auth0_id)
         user.aws_create_role()
 
-        expected_role_name = f"test_user_{username}"
+        expected_role_name = f"test_user_{auth0_id}"
```

Comment:
As said, the IAM role name shouldn't change. It'll have to be generated from the (GitHub) username as before to maintain compatibility.

[Also, Auth0's user ID includes a pipe (`|`) which is [not a valid character](http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_iam-limits.html) for an IAM name.]